lamindb/core/__init__.py

```diff
@@ -81,18 +81,18 @@
     ValidateFields,
 )
 
-from lamindb._curate import (
+from lamindb._query_manager import QueryManager
+from lamindb._query_set import QuerySet, RecordList
+from lamindb.core._feature_manager import FeatureManager, ParamManager
+from lamindb.core._label_manager import LabelManager
+from lamindb.curators import (
     AnnDataCurator,
```

Comment:
Ah, do we really want to re-export all curators at the root-level? That's going to clutter the root API because they don't even share a common prefix.

And `CurateLookup` and `BaseCurator` are not even user-facing interfaces.

I'd have the user-facing API under `curators`.